torch/utils/checkpoint.py

```diff
@@ -408,6 +413,10 @@ def checkpoint(
             context managers. The function and its recomputation will be run
             under the first and second context managers respectively.
             This argument is only supported if ``use_reentrant=False``.
+            This is useful e.g. for implementing selective checkpointing.
+            When selective checkpointing is used with torch.compile,
+            random ops (e.g. ``torch.dropout``) and in-place ops (e.g. ``torch.relu_``)
```

Comment:
Just interested - why dont we allow in place ops with SAC?

Reply:
Currently running into gradient mismatch for the inplace test case we have, will wait for Brian's stack to land and then investigate this again